lib/src/10_auth/auth_screen.dart

```diff
@@ -81,6 +85,26 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     return widget.phrases[phrase] ?? message;
   }
 
+  @override
+  void dispose() {
+    _emailNode.dispose();
+    _passwordNode.dispose();
+    super.dispose();
+  }
+
+  // The following override takes care of "setState() called after dispose()" exception.
+  // This exception is raised because when Enter key is pressed, it calls submit().
+  // This causes the Future to complete after the widget has already been disposed.
+  // More details can be found at the given link:
+  // "https://github.com/Norbert515/flutter_villains/issues/8".
+  
```

Comment:
One more request. Sorry for this. Can you move the comment into the override, right before if, and change 'The following override...' to 'This check...'

It is to make it clear where the comment relates to.